Examples/Examples/Cart/CartViewController.m

```diff
@@ -124,6 +123,16 @@ - (void)reloadData {
     [self.tableView reloadData];
 }
 
+- (void)startAnimating {
+    [self.activityIndicator startAnimating];
+    self.view.userInteractionEnabled = false;
```

Comment:
Should be `NO` to align with `YES`

Reply:
Interesting. I also change the code I copied.